### ISSUE TITLE
Age field and minor typo

### ISSUE THREAD

**Issue by tsuzumiC** [richardwilkes/gcs#247]
Would it be possible to make the Age field to accept text instead of just numbers? It would help characters that have a visual age and a actual age. Like "74 (Looks like 21)" or just 74(21).

Also, Intimidation skill in Basic Set Skills defaults to Perception instead of Will.

**Comment by richardwilkes** [comment]
Hmm... B202 Shows Intimidation defaulting from Will-5 or Acting-3, which is what the master library has. Are you looking somewhere else?

**Comment by tsuzumiC** [comment]
Picture probably helps better here.
https://puu.sh/Gi92Q/a42e474f5a.png

**Comment by richardwilkes** [comment]
Yes, that’s definitely wrong, but also not what is in the current master library. Was that from your sheet? If so, you need to update it with data from the library. GCS won’t do that for you, at least not yet.

**Comment by tsuzumiC** [comment]
No. That was not from my sheet. It is from the master library and it is still there for me when I updated to 2.0.2 (which should be latest?) As you can see the fields are grayed out.
https://puu.sh/GjxNy/40348ef464.png

**Comment by richardwilkes** [comment]
Hmm... I just went and looked again and my copy is also showing Perception - 5 -- I must have been looking at another entry somewhere. Anyway, easy enough to fix. Thanks for pointing it out.
